### DISCUSSION TITLE
Citation request

### DISCUSSION THREAD

**Discussion by BrijBoda** [sebastian-hofstaetter/teaching#128] [General]
Hello Community. 
I am writing my master's thesis on information retrieval topic and I would like to cite some of the information from the lectures. Can anyoun provide the original sources or guide me how to cite in this case?

**Comment by sebastian-hofstaetter** [comment]
Hi, thats cool! All lectures here are original material created by me and @sophiaalthammer, with participation from our students. All referenced materials are linked on each slide. If you want to cite our lectures you can do as follows:
Advanced Information Retrieval Course 2022, Hofstätter and Althammer 2022. https://github.com/sebastian-hofstaetter/teaching
Good luck with your masters thesis :)
